Felienne/hedy
# [FEATURE] Shows "last-used" on admin class overview, sort by date

**Description**
In this PR we improve the admin class overview by adding a new column; "last-used". This value is based on the activity of the teacher and gives insights in the classes activities within Hedy. We also automatically sort the classes by "last-used" date.

**Fix for**
This PR fixes #1882.

**How to test**
Make sure you are an admin and navigate to `/admin`. Notice the new column on the class overview page and the automatic sorting from recent to older.
